.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -0,0 +1,50 @@
+---
+name: 🚀 Feature Request
+about: I have a suggestion (might want to implement it myself 🙂)!
+labels: enhancement
+---
+
+<!--- Please keep this note for the community --->
+
+### Community Note
+
+* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
+* If you are interested in working on this issue or have submitted a pull request, please leave a comment
+
+<!--- Thank you for keeping this note for the community --->
+
+### Description
+<!--- Please specify additional labels if applicable to the issue else leave blank. --->
+* additional labels: [ ]
+(new_module/new_plugin/documentation/feature)
```

Comment:
I think we should just tell them to add the labels to the issue not in the text.